src/util/find_uid.c

```diff
@@ -90,8 +90,15 @@ static errno_t get_uid_from_pid(const pid_t pid, uid_t *uid)
                       path);
             return EOK;
         }
+        // case of hidepid=1 mount option for /proc
```

Comment:
We don't use the single-line comments in sssd. Can you change this to `/* ... */`